src/components/Link.react.js

```diff
@@ -47,7 +47,7 @@ export default class Link extends Component {
             window.location.pathname = href;
         } else {
             window.history.pushState({}, '', href);
-            window.dispatchEvent(new CustomEvent('onpushstate'));
+            window.dispatchEvent(new CustomEvent('_dashprivate_onpushstate'));
```

Comment:
Make the link <--> location event explicitly private. No one outside of DCC should listen to this.